models/migrations/v1_20/v255.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_20 //nolint
+
+import (
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/xorm"
+)
+
+func AddArchivedUnixToRepository(x *xorm.Engine) error {
+	type Repository struct {
+		ArchivedUnix timeutil.TimeStamp `xorm:"Default 0"`
```

Comment:
```suggestion
		ArchivedUnix timeutil.TimeStamp `xorm:"DEFAULT 0"`
```
Non-caps-lock xorm commands scare me.